gxtest/scissor.cpp

```diff
@@ -0,0 +1,581 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
```

Comment:
Other files in hwtests are currently GPLv2 only; the license migration that was done with main Dolphin doesn't seem to have been done for hwtests.  I've chosen to use GPLv2+ here; I don't think there are any problems with offering GPLv2+ for a specific file in a combined work that's currently GPLv2.

Reply:
Feel free to relicense any of the code I authored in the hwtests repository to GPLv2+ as well.

Reply:
@Pokechu22 Sounds good. But for consistency with the main repo, I would suggest using `SPDX-License-Identifier` instead of the old-style license comment.